application/templates/cronjob.yaml

```diff
@@ -54,7 +54,7 @@ spec:
           annotations: {{ toYaml . | nindent 12 }}
           {{- end }}
         spec:
-          {{- if $.Values.rbac.enabled }}
+          {{- if $.Values.rbac.serviceAccount.enabled }}
```

Comment:
Shouldn't we still keep the `if $.Values.rbac.enabled` line? If RBAC for this chart has been completely disabled, it shouldn't in theory matter if `rbac.serviceAccount.enabled` is true, is my thinking.

Reply:
Same goes for the deployment, then.
I didn't check if there are other parts as well, which would be equally affected.
I'm just building on your reasoning for the deployment :)